src/core/crypto.c

```diff
@@ -1880,6 +1880,22 @@ QuicCryptoProcessData(
                 Connection->Paths[0].Binding,
                 Connection,
                 &Info);
+
+            if (!Connection->State.HandleClosed &&
```

Comment:
Would appreciate a second pair of eyes on this. I'm not sure if this is the best way to tell that the connection has accepted.

Reply:
I think this should be fine.